### DISCUSSION TITLE
Universal Widget: Prüfung true/false geht nicht

### DISCUSSION THREAD

**Discussion by ppb1308** [inventwo/ioBroker.vis-inventwo#333] [Q&A]
Hallo,
im Universal Widget funktioniert für Widget Type = State die Prüfung auf eine Objekt ID (boolean) die true als Wert enthält nicht:

1. Fall
Typ = bolean, Wert falsch = leer, Wert wahr = leer, Wert = leer, Inhalt Objekt ID = true -> Anzeige Widget Werte für falsch
2. Fall
Typ = value, Wert falsch = false, Wert wahr = true, Wert = true, Inhalt Objekt ID = true -> Anzeige Widget Werte für falsch

Was kann ich ggf. anders einstellen?
Die Prüfung der gleichen Objekt-ID im Multi Widget klappt problemlos und korrekt.

Gruß,
Peter

**Comment by jkvarel** [comment]
Hallo,
falls das Problem noch besteht, Type State sendet nur einen Wert an den Dp. Du musst entweder Switch nehmen oder das Multi-Widget mit Type State. Beim Multi-Widget kannst du mehrere Zustände definieren

**Comment by Hans-Ju** [comment]
Hallo, ich möchte einen Status anzeigen. Hierfür habe ich ein Universal / State eingestellt mit unterschiedlichen Bildern und Texten für true und false. Leider wird, der Status des Datenpunktes nicht ausgewertet. Wenn ich auf Switch umstelle funktioniert alles bestens, leider könnte ich dann den Status auch auf dem Tablet per Klick umstellen. Das möchte ich jedoch nicht. Es soll lediglich den Status anzeigen. Wie oben beschrieben habe ich ein Multi / State eingerichtet. Damit komme ich leider überhaupt nicht klar. Nur eingeschränkte Text Möglichkeit und nur wenn ich darauf klicke. Danach geht gar nichts mehr. Wo ist mein Denkfehler? Viele Grüße Hans-Jürgen

**Comment by Hans-Ju** [comment]
Ich habe jetzt eine Lösung gefunden. Ich stelle den Universal auf Background.

**Reply by skvarel** [reply]
Hallo.

Richtig. Als 'Background' kann das Widget nur anzeigen aber nicht schalten.
Ein Type 'State' wäre ein einfacher Taster und eine 'Switch' ist ein Umschalter

Gruß, Sven.
